ISSUE_TEMPLATE.md

```diff
@@ -36,7 +36,7 @@ Stage Checklists
 ### Presentation Phase/Night
 - [ ] Move card to Pitches/Presentation list in waffle if you are presenting tonight
 - [ ] Present to group (3 min limit)
-- [ ] Group Q&A
+- [ ] Group Q&A (***JJB: Recommend we explicitly timebox this***)
```

Comment:
Yes, I was thinking something along those lines... and if the pomodoro can be used for a deeper dive amongst interested parties.